fair_mango/metrics/metrics.py

```diff
@@ -249,6 +712,72 @@ def __call__(self) -> tuple[Sequence, list]:
 
 
 class DemographicParityDifference(FairnessMetricDifference):
+    """Calculate Demographic Parity Fairness Metric using "difference" to
+    calculate the disparity between the different demographic groups present
+    in the sensitive feature.
+
+    Demographic Parity calculates the "difference" in the Selection Rate in the
```

Comment:
same as for confusionmatrix and performance matrix. You should have a mention of something like "for different demographic groups"